src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -92,6 +107,13 @@ public class SupportAction implements RootAction, StaplerProxy {
      */
     private final Logger logger = Logger.getLogger(SupportAction.class.getName());
 
+    private static final String SUPPORT_BUNDLE_FILE_NAME = "support-bundle.zip";
+    private static final String SUPPORT_BUNDLE_CREATION_FOLDER = Paths.get(System.getProperty("java.io.tmpdir"))
+            .resolve("support-bundle")
+            .toString();
```

Comment:
the system property `java.io.tmpdir` is always expected to exist? or could lead to NPE?

Reply:
it picks default. sorry didn't know

Reply:
Probably more straight to the point as
```
Files.createTempDirectory("support-bundle")
```

Reply:
Unfortunately not. See description at https://github.com/jenkinsci/jenkins-test-harness/pull/838.